### ISSUE TITLE
PDF from acmart vs eproofs from ACM

### ISSUE THREAD

**Issue by rionda** [borisveytsman/acmart#467]
There seems to be quite a few differences between the PDF produced by acmart and that from the eproofs one receives from ACM (or the service they use) once the paper is accepted.

* The positioning of the "acknowledgments" (the content of the `acks` environment) differs: in acmart, they are at the end of the paper, usually before the bibliography, while in the eproofs they are on the front page, in the same section as the Author's addresses. 
(I must admit that the above is at least true for funding information. My acks did not include acks to other researchers or reviewers, so I don't know where those would be).

* `\titlenote` does not add an asterisk at the end of the title in the eproofs, and the positioning of its content is slightly different (there is no "short" horizontal line above it, and it is inserted just above the Authors' addresses, after the "long" horizontal line).

* The name of the journal is spelled fully in the eproofs (in my case "ACM Transactions on Knowledge Discovery from Data") rather than abbreviated ("ACM Trans. Knowl. Discov. Data")

* The style for `itemize` seems different (dashes in place of bullets, and the indent seems different).

There are probably more that I didn't spot.

Should `acmart` be more in line with the eproofs (or vice versa)?

I also don't get why the eproofs preparers take the liberty of changing the running heads (by shortening them), when the `acmart` PDF shows that the full running heads would fit perfectly fine.  But I assume that's not something that `acmart` can fix.

**Comment by stephennspencer** [comment]
When you say "eproof" are you speaking of the HTML5 version of the article that TAPS generates from your LaTeX or Word source?

**Comment by rionda** [comment]
No, I'm talking about the PDF that one gets from (the service used by) ACM before a journal article is published, to check that everything looks correct, submit corrections, and possibly reply to queries from the publisher.

**Comment by stephennspencer** [comment]
I withdraw my comment, then. I should have picked up on the fact that you were speaking of a journal article.

**Comment by borisveytsman** [comment]
I think this is for the ACM to reslove...

**Comment by stephennspencer** [comment]
Matteo, are you sure you are using the same version of the LaTeX template file?

The TAPS-generated PDF shows the "acmart" version in the document properties section of the PDF. 

I'm still a little unclear as to what you're trying to compare: are you building your PDF locally, and comparing it to a PDF generated by TAPS, or something else?

**Comment by borisveytsman** [comment]
Cannot reproduce
